GETTING_STARTED.md

```diff
@@ -0,0 +1,63 @@
+# Getting Started as a Boutiques Developer
+
+This guide will help you set up your local environment for coding and testing Boutiques.
+
+## 0. Things you'll need
+- **A Linux system**. It _may_ be possible to develop Boutiques on Windows, but will likely cause many headaches.
+- **Root (sudo) access**. Unless your system already has everything you need installed, you'll need root permission to install new programs.
+- **Python**. Boutiques supports any version >= 2.7. It's a good idea to have multiple versions of Python installed (e.g. 2.7 and 3.7) so you can test your code for version compatibility.
+- **pip**. Most versions of Python will come with it, but in case you don't have it, instructions to install it are [here](https://pip.pypa.io/en/stable/installing/).
+- **Docker**. If you don't already have Docker, follow the [instructions](https://docs.docker.com/install/overview/) to install Docker CE.
+- **Singularity**. Follow the instructions [here](https://singularity.lbl.gov/install-linux) to download the latest stable release of Singularity.
+- **Git**. If you don't have Git, instructions to install it are [here](https://git-scm.com/download/linux).
+
+## 1. Clone the boutiques repository
+- Fork the [Boutiques repository](https://github.com/boutiques/boutiques).
+- Clone your fork locally: 
+    `git clone https://github.com/<YOUR-GITHUB-USERNAME>/boutiques.git`
+
+## 2. Create a virtual environment
+- Install virtualenv:
+  `sudo pip install virtualenv`
+- Create a virtual envrionment in the Python tools root directory:
+  `cd boutiques/tools/python`
+  `virtualenv env`
+- Use the environment whenever you work on Boutiques by running:
+  `source env/bin/activate`

```

Comment:
Given the above comment, this line would switch to `source env/boutiques/bin/activate`.